frontend/src/sass/components/_footer.scss

```diff
@@ -1,4 +1,4 @@
-.footer {
+.global-footer {
 	background: $lighter-black-colour;
 	
 	&__list {
```

Comment:
There is a mic because different people :D I will merge this and add in editorconfig so we can try to synchronise and can raise a PR to fix all file styles in one go later on.